### DISCUSSION TITLE
Handlebars/Mustache Highlighing?

### DISCUSSION THREAD

**Discussion by mkastner** [LazyVim/LazyVim#2006] [General]
How can I get Handlebars and Mustache code highlighting? LazyVim extras doesn't provide an option. Maybe I just need to add the hbs extension to treesitter somewhere.

BuI i have no clue how to do that in LazyVim.

**Comment by dpetka2001** [comment]
It seems that the feature you want is provided by the `glimmer` [parser](https://github.com/search?q=repo%3Anvim-treesitter%2Fnvim-treesitter%20handlebars&type=code). You could do something like 
```lua
return {
  {
    "nvim-treesitter/nvim-treesitter",
    opts = function(_, opts)
      vim.list_extend(opts.ensure_installed, {
        "glimmer",
      })
    end,
  }
}
```
in a file in `~/.config/nvim/lua/plugins/treesitter.lua` and then restart Neovim for the parser to be installed. This is to make the change permanent in the Lazyvim configuration in case you version control your dotfiles and want the same configuration across multiple computers or you want to restore current configuration from backup. You could also achieve the same by just doing `:TSInstall glimmer`, but that will only install the `glimmer` parser for your current Neovim configuration and you will have to do the same again on another computer if you want the same behavior.

**Comment by mkastner** [comment]
This is absolutely awesome! I like the way plugins work in LazyVim. And your suggested code snipped works just like that. Thanks for a great start this morning!
